moveit_planners/pilz_industrial_motion_planner/test/unit_tests/src/unittest_cartesian_limits_aggregator.cpp

```diff
@@ -32,63 +32,63 @@
  *  POSSIBILITY OF SUCH DAMAGE.
```

Comment:
This test seems somewhat obsolete with the new changes, no? Please, either comment with a TODO or remove.

Reply:
I have removed the tests